theme buttons as per Joan

mateosalta opened the pull request:
- Primary action on top
- neutral secondary action for format (it displays confirmation, not immediate destruction)

@cibersheep (design review needed :)

mateosalta committed:
theme buttons as per Joan

cibersheep commented:
Looks good to me!
Very good color police ;)

UniversalSuperBox commented:
Here's some comparisons. New on the left, old on the right.

![confirmremove](https://user-images.githubusercontent.com/21966173/68245473-a05f5100-ffdc-11e9-8008-480af088edb4.png)

![format](https://user-images.githubusercontent.com/21966173/68245481-a35a4180-ffdc-11e9-9cb6-1ad7eb3087b4.png)

![formatcomplete](https://user-images.githubusercontent.com/21966173/68245493-a7865f00-ffdc-11e9-8bbd-85008ecafe37.png)

![safetoremove](https://user-images.githubusercontent.com/21966173/68245506-ad7c4000-ffdc-11e9-9ab5-43704b6fe5c7.png)

![unmounterror](https://user-images.githubusercontent.com/21966173/68245523-b10fc700-ffdc-11e9-8957-91c7ba476534.png)

cibersheep commented:
Excellent. Thanks you.
The only thing I see is that ok button for the «Format Complete» shouldn't be red :)

Somewhere here? https://github.com/ubports/ciborium/blob/xenial/share/ciborium/qml/components/FormatDialog.qml#L87

cibersheep commented on share/ciborium/qml/components/FormatDialog.qml:
```diff
@@ -6,18 +6,10 @@ Dialog {
         id: okBtn
         text: i18n.tr("Continue with format")
-        color: UbuntuColors.orange
+        color: UbuntuColors.red
```
This stays red when format is completed

mateosalta commented:
oh, interesting. Should "Unmount Error" say Ok instead of Continue - and maybe be a neutral color?

cibersheep commented:
Well spotted. So, yes: We have two tweaks here.

1) Format Completed: ok -> green / blue
![image](https://user-images.githubusercontent.com/6640041/68312182-f6062d00-00b2-11ea-86cc-f75d939b58ec.png)

2) Umount Error: continue -> ok in uncolored / blue
![image](https://user-images.githubusercontent.com/6640041/68312242-0e764780-00b3-11ea-8149-0fd550ef9186.png)

mateosalta committed:
Update FormatDialog.qml

theme.palette.normal colors, and make sure success/failure buttons are themed proper neutral on failure positive on success

mateosalta committed:
Update SafeRemoval.qml

update colors

mateosalta commented:
ok, errors should now have a neutral button ( no color, action is not positive - but button just closes message)

and successfully completed now green/positive

I also bumped the popup to 1.3 - this supports theming
![screenshot20191106_185802041](https://user-images.githubusercontent.com/1965279/68351016-d3324380-00c7-11ea-9314-4aa00722cd8d.png)

mateosalta commented:
huh, what happened to the build... didn't change much :0

mateosalta committed:
try build again?

mateosalta commented:
@UniversalSuperBox yay, it built. probably temporary problem :)

UniversalSuperBox reviewed (changes requested):
Something appears to have gone wrong:

```
2019/11/20 14:15:58 log.go:88: main.qml.:40: Safe removal button clicked
2019/11/20 14:15:58 log.go:88: .:0: QQmlComponent: Component is not ready
2019/11/20 14:15:58 log.go:88: popupUtils.js:50: file:////usr/share/ciborium/qml/components/SafeRemoval.qml:120 Expected token `:'
2019/11/20 14:15:58 log.go:88: popupUtils.js:51: PopupUtils.open(): Failed to create the popup object.
```

mateosalta commented on share/ciborium/qml/components/SafeRemoval.qml:
```diff
@@ -115,6 +115,8 @@ Dialog {
                 explicit: true
                 visible: true
+                text: i18n.tr("Ok")
+                theme.palette.normal.overlaySecondaryText
```
whops :0

mateosalta committed:
add in color:

mateosalta reviewed (commented):
ok, looks like i missed "color:"  in: line 119 share/ciborium/qml/components/SafeRemoval.qml
